backend/api/views/book.py

```diff
@@ -184,3 +184,30 @@ def delete_quiz(user_id):
     return create_response(
         message="Successfully deleted book", status=200, data={"status": "success"}
     )
+
+
+@book.route("/edit_book", methods=["POST"])
+@admin_route
+def edit_book(user_id):
+    user_data = request.get_json()
+    if invalid_book_data(user_data):
+        return create_response(
+            message="Missing required book info", status=422, data={"status": "fail"}
+        )
+
+    book_to_edit = Book.query.get(user_data["book_id"])
+    if book_to_edit is None:
+        return create_response(
+            message="Book not found", status=422, data={"status": "fail"}
+        )
+
+    book_to_edit.name = (user_data["name"],)
```

Comment:
interesting syntax just curious why is do we wrap the name in a tuple?

Reply:
this was an accident